paramak/parametric_neutronics/neutronics_model_from_reactor.py

```diff
@@ -101,11 +101,12 @@ def __init__(
 
         # Only 360 degree models are supported for now as reflecting surfaces
         # are needed for sector models and they are not currently supported
-        if reactor.rotation_angle != 360:
-            reactor.rotation_angle = 360
-            print('remaking reactor as it was not set to 360 degrees')
-            reactor.solid
-            # TODO make use of reactor.create_solids() here
+        if hasattr(reactor, 'rotation_angle'):
+            if reactor.rotation_angle != 360:
+                reactor.rotation_angle = 360
+                print('remaking reactor as it was not set to 360 degrees')
+                reactor.solid
```

Comment:
Is this needed ? If .solid is called elsewhere, since the hashvalue is changed, it should be remade anyway. This could save some time.

Reply:
ah yes good point, I was just copying what was there before without thinking. Thanks for spotting that